README.md

```diff
@@ -14,26 +14,61 @@ VennDiagram is a R package for generating high-resolution, customizable Venn dia
+- **Row 3, column 2**: a three-set Euler diagram where one set is totally included in another set, which is itself totally included in the third set.
+- **Row 3, column 2**: a three-set Euler diagram showing three distinct sets.
 
 ![](https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-12-35/MediaObjects/12859_2010_Article_5075_Fig2_HTML.jpg?as=webp)
 
-**Selected Venn diagram special cases and Euler diagrams drawn by the VennDiagram package.** Row 1, column 1: automatically drawn, customizable lines that optimize display of partial areas when individual partial areas become too small in two-set Venn diagrams. Row 1, column 2: a two-set Euler diagram showing total inclusion of one of the sets. Row 1, column 3: a two-set Euler diagram showing two distinct sets. Row 2, column 1: a three-set Euler diagram where one set has no discrete elements. Row 2, column 2: a three-set Euler diagram where one set has no discrete elements is totally included in one of the other two sets. Row 2, column 3: a three-set Euler diagram where two sets have no discrete elements and are included in a larger third set. Row 3, column 1: a three-set Euler diagram showing total inclusion of two sets that are distinct from the third set. Row 3, column 2: a three-set Euler diagram where one set is totally included in another set, which is itself totally included in the third set. Row 3, column 2: a three-set Euler diagram showing three distinct sets.
+A side-by-side comparison of an Euler diagram and a Venn diagram for the same hypothetical sets.
 
-![](https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-12-35/MediaObjects/12859_2010_Article_5075_Fig3_HTML.jpg?as=webp)
+<!-- to get A, B -->
+<ol type="a">
+    <li>The Euler diagram shows only non-zero areas and can therefore be more graphically accurate.</li>
+    <li>The Venn diagram shows the non-existent area as an area with zero content. Though this is not graphically accurate, it preserves the recognisability of a Venn diagram.</li>
+</ol>
 
-**A side-by-side comparison of an Euler diagram and a Venn diagram for the same hypothetical sets.** A) The Euler diagram shows only non-zero areas and can therefore be more graphically accurate. B) The Venn diagram shows the non-existent area as an area with zero content. Though this is not graphically accurate, it preserves the recognisability of a Venn diagram.
+![](https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-12-35/MediaObjects/12859_2010_Article_5075_Fig3_HTML.jpg?as=webp)
 
 ## Installation
 
 To install the latest public release of VennDiagram from CRAN:
-```
+
+```R
 install.packages("VennDiagram")
 ```
+
 Or to install the latest development version from Github:
-```
+
+```R
 # install.packages("devtools")
 
 devtools::install_github("uclahs-cds/public-R-VennDiagram")
```

Comment:
Move this up above the example. In the TOC it is listed as 2nd which makes sense.

Reply:
Good catch, moved it back up!